pyramid_oauthlib/__init__.py

```diff
@@ -21,7 +21,20 @@
     'redirect_uri',
     'scope',
     'state',
+    'user'
```

Comment:
I think we might be missing `token` and `token_type_hint` here, still.

Reply:
It looks like they also have `scopes`. So many! Sorry!

Reply:
Yikes, and `redirect_uri`. Maybe we just copy from here: https://github.com/oauthlib/oauthlib/blob/v2.1.0/oauthlib/common.py#L396

😆

Reply:
Done and done